ng/src/web/pages/groups/component.js

```diff
@@ -39,36 +40,41 @@ class GroupComponent extends React.Component {
   constructor(...args) {
     super(...args);
 
+    this.state = {dialogVisible: false};
+
+    this.closeGroupDialog = this.closeGroupDialog.bind(this);
     this.openGroupDialog = this.openGroupDialog.bind(this);
   }
 
   openGroupDialog(group) {
     const {gmp} = this.props;
 
+    let allUsers = [];
+    gmp.users.getAll().then(response => {
+      allUsers = response.data;
+      this.setState({allUsers});
+    });
+
     if (is_defined(group)) {
-      const {
-        id,
-        name,
-        comment,
-        users,
-      } = group;
-
-      this.dialog.show({
-        id,
-        name,
-        comment,
-        users,
-      }, {
-        title: _('Edit Group {{name}}', group),
+
+      const title = _('Edit Group {{name}}', group);
+
+      this.setState({
+        dialogVisible: true,
+        group,
+        title,
```

Comment:
I (re-)thought about allUsers and we have to ensure the state doesn't contain any old and potentially protected data. Therefore we should ensure that allUsers get overriden in the state when opening the dialog. Therefore we should add it to the state here and below too. Then it will either contain the correct data for the current user or an empty array which will get filled later.